ceph.spec.in

```diff
@@ -733,6 +733,10 @@ Group:		Development/Libraries/Python
 Requires:	librgw2 = %{_epoch_prefix}%{version}-%{release}
 Requires:	python%{python3_pkgversion}-rados = %{_epoch_prefix}%{version}-%{release}
 Provides: 	python3-rgw = %{_epoch_prefix}%{version}-%{release}
+%if 0%{without python2}
+Provides:	python-rgw = %{_epoch_prefix}%{version}-%{release}
```

Comment:
i think this actually complies https://fedoraproject.org/wiki/User:Churchyard/Packaging:Python#The_.25python_provide_macro . if the "system version of python" of the targeting distro is python3, we should do this. i guess this applies to "0%{?fedora} >= 29 || 0%{?suse_version} >= 1500 || 0%{?rhel} >= 8".